dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -515,7 +517,7 @@ portalRouter.route('/facilities/:id/status').put(updateFacilityStatusController.
  * /facilities/:facilityId/amendments/:amendmentId:
  *   get:
  *     summary: Get a Portal GEF facility amendment
- *     tags: [Portal - GEF]
+ *     tags: [Portal - Amendments]
```

Comment:
```suggestion
 *     tags: [Portal - GEF Amendments]
```

Reply:
These endpoints are actually generic and could be used on BSS/EWCS deals if that functionality were added - I've removed all reference to `GEF` in the docs here

Reply:
Happy either way - only concern is that someone could think amendments are for both BSS/EWCS and GEF, which is not the case; But great that we have it ready for both product types.